api/v1alpha1/packagebundle_webhook.go

```diff
@@ -50,10 +55,18 @@ func (r *PackageBundle) ValidateDelete() error {
 }
 
 func (r *PackageBundle) validate() error {
-	valid, digest, yml, err := signature.ValidateSignature(r, signature.EksaDomain)
+	//TODO Turn this into fields in a new CRD used for signature validation configuration
```

Comment:
We need a new CRD? I thought this was going in PackageBundleController

Reply:
Ideally we'd use a new CRD and completely extract all the signature validation stuff into its own thing that can live without the eksa packages controller. That way people could use that for other types of validations. TBD

Now this may not get much adoption since we're also planning to provide that functionality via OPA.